nix/default.nix

```diff
@@ -139,6 +142,7 @@ let
         # sandbox, so we can just use node instead
         # (idea taken from `plutus-playground-client`)
         checkPhase = ''
+          export OGMIOS_FIXTURES="${buildOgmiosFixtures { }}"
```

Comment:
Again, this should be removed as it requires users to build the fixtures for every test derivation created with this library. This is only useful for our own unit test, so we need to override the `checkPhase` instead of that single derivation

Starting at line 427 of the flake, you can change `ctl-unit-test` to:

```
            ctl-unit-test = (project.runPursTest {
              testMain = "Test.Unit";
              sources = [ "src" "test" "fixtures" ];
            }).overrideAttrs(oas: {
              checkPhase = ''       
                export OGMIOS_FIXTURES="${project.buildOgmiosFixtures { }}"
              '' + oas.checkPhase;
            });
```